Slim/Networking/Async/HTTP.pm

```diff
@@ -122,8 +127,8 @@ sub new_socket {
 			my %args = @_;
 
 			# Failed. Try again with an explicit SNI.
-			$args{SSL_hostname} = $args{Host};
-			$args{SSL_verify_mode} = Net::SSLeay::VERIFY_NONE() if $prefs->get('insecureHTTPS');
+			$args{SSL_hostname} = $args{Host} unless defined $args{SSL_hostname};
+			$args{SSL_verify_mode} = Net::SSLeay::VERIFY_NONE() if $prefs->get('insecureHTTPS') && !defined $args{SSL_verify_mode};
```

Comment:
Can't you use `$args{SSL_hostname} //= ... ` style here, instead of using an explicit condition?

Reply:
I'd like to but last time you kicked me for using it because I think it was not there I think in 5.08 or os

Reply:
I believe you reminded me of it with the transcoding PR, where you used it earlier today ;-). It's fine, Perl 5.8 is no longer supported.

Reply:
ok, will change that then